### PR TITLE
split docker composes into two

### PR DESCRIPTION
**Pull request by abarkellumich**

### CODE REVIEW

File: ara/alert-docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    logging:
+      options:
+        max-size: "10m"
+        max-file: "3"
+    networks:
+      local:
+        aliases:
+          - elastalert
+
+networks:
```
Comment (rlreamy) [inline ara/alert-docker-compose.yml:22]: I think that the main docker-compose needs to own the network, and this one needs to join it.

The code to join a network looks like this, where "dataLake" is the name of the existing network to join:
```
networks:
  dataLake:
    external: true
```

**Review by rlreamy — CHANGES_REQUESTED** [review]
See the notes in alert-docker-compose

File: ara/alert-docker-compose.yml (hunk 1 above)
Comment (rlreamy) [reply ara/alert-docker-compose.yml:22]: In order for this to work, we need to set up a network named logAggregator in the other docker-compose file that we can join.

Comment (abarkellumich) [reply ara/alert-docker-compose.yml:22]: We do have one, is there something I'm missing that needs to be on it to open it up for external connections??


`networks:
  local:
    name: logAggregator
    driver: bridge
    ipam:
      driver: default
      config:
        - 
          subnet: 10.3.0.0/16`

Comment (abarkellumich) [reply ara/alert-docker-compose.yml:22]: Oh, that that didn't paste in very nicely, but see bottom of the the docker-compose.yml file

Comment (rlreamy) [reply ara/alert-docker-compose.yml:22]: Ok...I was thinking that didn't exist yet, and didn't see it as a change. Thanks

**Review by rlreamy — APPROVED** [review]

**Pull request merged**